docs/develop/api-reference/index.mdx

```diff
@@ -43,7 +43,7 @@ Attributes:
 
 ### List caches
 
-Lists all caches for the provided auth token.
+Lists all caches for the provided Momento API token.
```

Comment:
TBH here seems like maybe just drop the part about auth token. We dont use on other api reference. For ex "Delete cache named foo for given API token" sounds weird also to me.

```suggestion
Lists all caches.
```